src/shared/lib/services/collectImageUrls.test.ts

```diff
@@ -0,0 +1,578 @@
+      name: 'Default',
+      logo: 'https://media-dev.cmiml.net/mindlogger/content/some-logo.png',
+      backgroundImage:
+        'https://media-dev.cmiml.net/mindlogger/content/some-backgroundImage.png',
+      primaryColor: '#0067a0',
+      secondaryColor: '#fff',
+      tertiaryColor: '#404040',
+    },
+    activities: [
+      {
+        id: '52ee0b2e-9e9a-4145-9b2f-2ca7ac5a66ad',
+        name: 'Activity0000 (1)',
+        description: 'ASASD',
+        image:
+          'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/c22ba829-ad08-4a7c-bbe6-9fd5a719f7e3/IMAGE%202.png',
+        isReviewable: false,
+        isSkippable: false,
+        showAllAtOnce: false,
+        isHidden: false,
+        responseIsEditable: true,
+        order: 1,
+        splashScreen:
+          'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/c26556ef-ef83-4782-9397-ea017a534e1c/%D0%A8%D0%AC%D0%A4%D0%9F%D0%A3%206.png',
+      },
+    ],
+    activityFlows: [],
+    encryption: {
+      publicKey: '[59]',
+      prime: '[123,91]',
+      base: '[2]',
+      accountId: '2923f4a9-20ef-4995-a340-251d96ff3082',
+    },
+    streamEnabled: false,
+  };
+
+  it('collectAppletRecordImageUrls should collect image URLs from AppletDto', () => {
+    const result = collectAppletRecordImageUrls(sampleApplet);
+    expect(result).toEqual([
+      'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/0e06365a-acc7-4db9-b7fc-8a0017a2354f/%D0%A8%D0%AC%D0%A4%D0%9F%D0%A3%206.png',
+      'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/68dff67f-47af-4ba9-adec-9440077b0e3d/IMAGE%202.png',
+      'https://media-dev.cmiml.net/mindlogger/content/some-logo.png',
+      'https://media-dev.cmiml.net/mindlogger/content/some-backgroundImage.png',
+    ]);
+  });
+
+  it('collectAppletDetailsImageUrls should collect image URLs from AppletDetailsDto', () => {
+    const result = collectAppletDetailsImageUrls(sampleAppletDetails);
+    expect(result).toEqual([
+      'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/0e06365a-acc7-4db9-b7fc-8a0017a2354f/%D0%A8%D0%AC%D0%A4%D0%9F%D0%A3%206.png',
+      'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/68dff67f-47af-4ba9-adec-9440077b0e3d/IMAGE%202.png',
+      'https://media-dev.cmiml.net/mindlogger/content/some-logo.png',
+      'https://media-dev.cmiml.net/mindlogger/content/some-backgroundImage.png',
+      'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/c26556ef-ef83-4782-9397-ea017a534e1c/%D0%A8%D0%AC%D0%A4%D0%9F%D0%A3%206.png',
+      'https://media-dev.cmiml.net/mindlogger/content/2923f4a9-20ef-4995-a340-251d96ff3082/c22ba829-ad08-4a7c-bbe6-9fd5a719f7e3/IMAGE%202.png',
+    ]);
+  });
+
+  it('collectActivityDetailsImageUrls should collect image URLs from ActivityDto', () => {
```

Comment:
Please remove "collectActivityDetailsImageUrls" word from string parameter of "it" functions.
When we call "it" function we should stick to the code style so that when we read the code line we build a sentence using the following pattern: it("should do something") / it("returns something").
Example:
<img width="701" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-app-refactor/assets/20073193/eabd121e-76bc-4ffd-a557-ff6f9235eb60">

Reply:
Please apply the same change to every new test case